integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/ConsumerMain.java

```diff
@@ -110,6 +113,9 @@ private static void testStandalone() throws Throwable {
     ITJUnitUtils.runWithHighwayAndRest(TestTrace.class);
     ITJUnitUtils.run(TestTraceEdge.class);
     ITJUnitUtils.runWithHighwayAndRest(TestParamCodec.class);
+    ITJUnitUtils.run(TestParamCodecEdge.class);
+
+    ITJUnitUtils.runWithTransports(Collections.singletonList(Const.RESTFUL), TestRestServerConfig.class);
```

Comment:
create a new method: runWithRest

Reply:
Fixed. I find there is already a runWithRest method in ITJunitUtils.